xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -651,8 +667,8 @@ For input data within a `ReadOnlySpan<byte>`, this always returns `false`. For i
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets the mode of this instance of the <see cref="T:System.Text.Json.Utf8JsonReader" />.</summary>
```

Comment:
What will this look like in Intellisense? Will the value tag be visible? With just reading the summary, it isn't clear what "mode" means.

If only the summary tag is visible, we may want to add more details:
```suggestion
        <summary>Gets the mode of this instance of the <see cref="T:System.Text.Json.Utf8JsonReader" /> which indicates whether all the JSON data was provided or there is more data to come.</summary>
```

Reply:
`<value>` is not shown in IntelliSense, I just tested it. I think this is only for the MS Docs website, just like remarks.

Reply:
We still need to fill it, otherwise the API will show up as undocumented.